### DISCUSSION TITLE
Making sample set from collection of .wav files

### DISCUSSION THREAD

**Discussion by tloose** [GrandOrgue/grandorgue#905] [Help]
I am missing something.  I collected some .wav files and created a .ODF around them that "works", but the samples play the full length - note off is ignored.

Can someone provide some insight?

**Comment by tloose** [comment]
Made no difference.

I also (recently) get an error message that each Pipe999 file contains no
pitch information.

On Thu, Dec 16, 2021 at 5:13 PM sandstone42 ***@***.***>
wrote:

> Does your ODF have a PipexxxLoadRelease=Y statement for each pipe (xxx is
> pipe number from 001 to 999?) If so, remove the statements from the ODF or
> change them all to PipexxxLoadRelease=N
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/GrandOrgue/grandorgue/discussions/905#discussioncomment-1829296>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AT34RNPAYUP7T362TOQKQ23URJXB5ANCNFSM5KHGNKEA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Tim

**Comment by tloose** [comment]
Do I need to apply some process to the wav files to make them suitable for
GrandOrgue?

On Thu, Dec 16, 2021 at 5:13 PM sandstone42 ***@***.***>
wrote:

> Does your ODF have a PipexxxLoadRelease=Y statement for each pipe (xxx is
> pipe number from 001 to 999?) If so, remove the statements from the ODF or
> change them all to PipexxxLoadRelease=N
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/GrandOrgue/grandorgue/discussions/905#discussioncomment-1829296>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AT34RNPAYUP7T362TOQKQ23URJXB5ANCNFSM5KHGNKEA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Tim

**Reply by sandstone42** [reply]
Did you use an application like Loop Auditioneer to make loops in the sample?

**Comment by tloose** [comment]
No, but I just found out about loopauditioner and am trying it now.

On Thu, Dec 16, 2021 at 9:37 PM sandstone42 ***@***.***>
wrote:

> Did you use an application like Loop Auditioneer to make loops in the
> sample?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/GrandOrgue/grandorgue/discussions/905#discussioncomment-1831247>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AT34RNNTAY5FNO25GLKRRELURKV7RANCNFSM5KHGNKEA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Tim

**Reply by sandstone42** [reply]
Also, make sure each stop (rank) element has an entry "Percussive=N" because that also can cause this situation.

**Comment by tloose** [comment]
Thanks for your help.
I used batch processing to create loops, and that seems to work for all but
one of the wav files.

I still need to figure out how to deal with warnings about release samples
and tuning information.

On Thu, Dec 16, 2021 at 9:47 PM sandstone42 ***@***.***>
wrote:

> Also, make sure each stop (rank) element has an entry "Percussive=N"
> because that also can cause this situation.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/GrandOrgue/grandorgue/discussions/905#discussioncomment-1831261>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AT34RNOHDO5EF4NFTHRIQNLURKXEZANCNFSM5KHGNKEA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Tim

**Reply by sandstone42** [reply]
There is a section for metadata in the WAV file format that allows storage of the MIDI note number. I think that is only required if you use alternate tunings but I'm not that familiar with the coding of Grandorgue. Loopauditioneer has a function to use what is called a Fast Fourier Transform to calculate the pitch of a WAV file and add the information in. Also, you can manually add a Cue point for the release sample.

**Comment by tloose** [comment]
I got the cue points entered and thus fixed the release point warning.

This has been a good learning experience for me; learning new things is the
spice of life :)

On Fri, Dec 17, 2021 at 10:02 AM sandstone42 ***@***.***>
wrote:

> There is a section for metadata in the WAV file format that allows storage
> of the MIDI note number. I think that is only required if you use alternate
> tunings but I'm not that familiar with the coding of Grandorgue.
> Loopauditioneer has a function to use what is called a Fast Fourier
> Transform to calculate the pitch of a WAV file and add the information in.
> Also, you can manually add a Cue point for the release sample.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/GrandOrgue/grandorgue/discussions/905#discussioncomment-1836631>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AT34RNN2LY7XV6Q2XVAMVOTURNNJLANCNFSM5KHGNKEA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Tim

**Reply by sandstone42** [reply]
You mentioned you used batch processing to create loops and that seemed to work for all but one of the wav files.
Was the sample that didn't work possibly a reed or noisy recording? You can relax the parameters on the loop finder. If you can get at least one loop that sounds fairly good you can then use the crossfade function to help blend the loop points for a smoother transition.

**Comment by tloose** [comment]
Good tip - thanks

On Mon, Dec 20, 2021 at 10:50 AM sandstone42 ***@***.***>
wrote:

> You mentioned you used batch processing to create loops and that seemed to
> work for all but one of the wav files.
> Was the sample that didn't work possibly a reed or noisy recording? You
> can relax the parameters on the loop finder. If you can get at least one
> loop that sounds fairly good you can then use the crossfade function to
> help blend the loop points for a smoother transition.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/GrandOrgue/grandorgue/discussions/905#discussioncomment-1848291>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AT34RNORW2ZEVEJJ342VLRDUR5NGVANCNFSM5KHGNKEA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Tim
